src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
```

Comment:
Model are rarely included into testing as constructors/getters/setters are naturally involved in other test classes and usually covered automatically without separate test class